waku/v2/node/wakunode2.nim

```diff
@@ -55,6 +55,9 @@ const clientId* = "Nimbus Waku v2 node"
 # Default topic
 const defaultTopic = "/waku/2/default-waku/proto"
 
+# Default Waku Filter Timeout
+const WakuFilterTimeout: Duration = chronos.days(1)
```

Comment:
I think you should be able to assign the const using simpler syntax, e.g.
`WakuFilterTimeout = 1.days` or similar